tools/grid/README.md

```diff
@@ -1,3 +1,52 @@
-# MOM6 grid generation tool
-* [ESMG gridtools](https://github.com/ESMG/gridtools)
-* [COSIMA regional-mom6](https://github.com/COSIMA/regional-mom6) 
+# MOM6 grid generation tool 
+
+  
+
+This documentation provides an overview of how we use ESMG gridtools and COSIMA-regional-mom6 gridtools for the MOM6 ocean model. These tools are essential for handling, manipulating, and visualizing grid data in MOM6 simulations. 
+
+  
+
+## [ESMG Gridtools](https://github.com/ESMG/gridtools) 
+
+  
+
+**ESMG (Earth System Modeling Grid) gridtools** are a set of utilities designed to facilitate the handling and manipulation of grid data used in Earth system models, including MOM6. 
+
+### Key Features 
+- **Grid Generation and Manipulation**: 
+  - Create various types of model grids (rectilinear, curvilinear, etc.). 
+  - Interpolate data between different grids. 
+  - Adjust grid resolutions. 
+    
+### Generate grids using ESMG 
+- **Set up conda environment**:  
+    - clone the repository from GitHub: 
+```bash 
+git clone https://github.com/ESMG/gridtools.git 
+``` 
+  - **Create a conda environment**: When creating the conda environemnt, issues might arise from incopatibility between environemnts. [This tutorial](https://github.com/ESMG/gridtools/blob/de0a18c1ce0807748aa70023300dfc415277bd4c/docs/conda/README.md?plain=1#L70) was very helpful providing alternative istallation methods. Here I use alternative installation method (2):  
```

Comment:
incompatibility?

Reply:
Here, I meant "incompatibility between packages." From my understanding, the GridTools Conda environment hasn't been consistently maintained, which can cause issues when installing the environment. However, the original tutorial provides alternative methods to help with the installation (see the section [here](https://github.com/ESMG/gridtools/blob/de0a18c1ce0807748aa70023300dfc415277bd4c/docs/conda/README.md?plain=1#L70)). I used method (2) and still had to manually install a few packages to make it work. Let me know if this explanation makes sense or if it is still too confusing.

Reply:
@gabyneg, thanks for your clarification. It is my bad that I did not make it clear in my original comment. Is 'incopatibility' a typo? If so, it should be 'incompatibility.

Reply:
Yes, I will fix that! thank you!